src/vm/prestub.cpp

```diff
@@ -1377,6 +1377,30 @@ PCODE MethodDesc::DoPrestub(MethodTable *pDispatchingMT)
             if (pModule->IsReadyToRun())
             {
                 pCode = pModule->GetReadyToRunInfo()->GetEntryPoint(this);
+#ifdef PROFILING_SUPPORTED
+                if (pCode != NULL)
+                {
+                    BOOL fShouldSearchCache = TRUE;
+                    {
+                        BEGIN_PIN_PROFILER(CORProfilerTrackCacheSearches());
+                        g_profControlBlock.pProfInterface->
+                            JITCachedFunctionSearchStarted((FunctionID) this,
+                                                           &fShouldSearchCache);
+                        END_PIN_PROFILER();
+                    }
+                    if (!fShouldSearchCache)
+                    {
+                        pCode = NULL;
+                    }
+                    else
+                    {
```

Comment:
Would it make more sense for the call to `pModule->GetReadyToRunInfo()->GetEntryPoint(this)` to be here?